packages/dialtone-vue2/components/filter_pill/filter_pill.vue

```diff
@@ -0,0 +1,322 @@
+<template>
+  <div
+    class="d-filter-pill"
+    data-qa="dt-filter-pill"
+  >
+    <dt-popover
+      v-model="isPopoverOpen"
+      :append-to="appendTo"
+      :fallback-placements="fallbackPlacements"
+      :max-height="maxHeight"
+      :max-width="maxWidth"
+      :modal="false"
+      :padding="padding"
+      :placement="placement"
+    >
+      <template #anchor>
+        <dt-button
+          v-dt-tooltip="alphaTooltipText"
+          :active="isActive"
+          :class="[
+            'd-filter-pill__primary',
+            {
+              'd-filter-pill--selected': isActive,
+              'd-filter-pill__primary--has-clear': hasClear,
+            },
+          ]"
+          :disabled="disabled"
+          :kind="buttonKind"
+          :size="size"
+          data-qa="dt-filter-pill__button"
+          icon-position="right"
+          importance="outlined"
+          @click="isPopoverOpen = true"
+        >
+          <span class="d-filter-pill__label">
+            <span class="d-filter-pill__label-alpha">{{ label }}</span>
```

Comment:
```suggestion
            <span class="d-filter-pill__label-alpha">
              <slot>{{ label }}</slot>
            </span>
```
See previous default slot comment.